submissions/RomanSaiko/memory-pair-game/scripts.js

```diff
@@ -0,0 +1,115 @@
+let cards = [
+    {
+        id: '0',
+        name: 'c.ronaldo',
+        path: 'img/c.ronaldo.png'
+    },
+    {
+        id: '1',
+        name: 'figo',
+        path: 'img/figo.png'
+    },
+    {
+        id: '2',
+        name: 'kaka',
+        path: 'img/kaka.png'
+    },
+    {
+        id: '3',
+        name: 'messi',
+        path: 'img/messi.png'
+    },
+    {
+        id: '4',
+        name: 'ronaldinho',
+        path: 'img/ronaldinho.png'
+    },
+    {
+        id: '5',
+        name: 'ronaldo',
+        path: 'img/ronaldo.png'
+    },
+    {
+        id: '6',
+        name: 'shevchenko',
+        path: 'img/shevchenko.png'
+    },
+    {
+        id: '7',
+        name: 'zidane',
+        path: 'img/zidane.png'
+    }
+]
+cards = cards.concat(cards)
+const cardsWrapper = document.getElementById('cards-wrapper')
+let amountOfMatches = 0
+let firstCard = null
+let cardItems = null
+
+const initCards = () => {
+    let cardsWrapperContent = ''
+    const cardsContent = cards.sort(() => 0.5 - Math.random())
+    cardsContent.forEach(item => {
+        cardsWrapperContent += `
+            <div class="flip-card" data-card="${item.id}">
+                <div class="flip-card-inner">
+                    <div class="flip-card-front"></div>
+                    <div class="flip-card-back">
+                        <img src="${item.path}" alt="${item.name}">
+                    </div>
+                </div>
+            </div>
+        `
+    })
+    cardsWrapper.innerHTML = cardsWrapperContent
+    cardItems = document.querySelectorAll('.flip-item')
+}
+
+const cardsWrapperHandler = ({target}) => {
```

Comment:
It's also a tricky moment with the naming. But it's clear that it's a handler.
So maybe `onCardsWrapperClick` would be better here.
It's up to you to fix it, but my teammates on my previous project were paying attention to it.